domain/dcsa_ovs_domain.yaml

```diff
@@ -261,7 +221,7 @@ components:
 #########################    
  
  ################################################
- ### Verfie locationType in the DCSA context ####
+ ### Verify locationType in the DCSA context ####
```

Comment:
We need to discuss the property location_type as it is used in the entier project multiple times with different matters